keras/legacy/interfaces.py

```diff
@@ -88,3 +89,16 @@ def raise_duplicate_arg_error(old_arg, new_arg):
 legacy_gaussiannoise_support = generate_legacy_interface(
     allowed_positional_args=['stddev'],
     conversions=[('sigma', 'stddev')])
+
+legacy_recurrent_support = generate_legacy_interface(
+    allowed_positional_args=['units'],
+    conversions=[('output_dim', 'units'),
+                 ('init', 'kernel_initializer'),
+                 ('inner_init', 'recurrent_initializer'),
+                 ('inner_activation', 'recurrent_activation'),
+                 ('forget_bias_init', 'bias_initializer'),
```

Comment:
We could consider passing an optional custom function to `generate_legacy_interface`, to preprocess the received args and kwargs. This would allow this kind of customization.